csharp/src/Ice/Current.cs

```diff
@@ -18,6 +19,10 @@ public class Current
         /// <summary>The <see cref="Connection"/> over which the request was dispatched.</summary>
         public Connection Connection { get; }
 
+        /// <summary>The request deadline. The peer sets the deadline to the absolute time at which its timeout will be
+        /// triggered. With Ice1 the peer doesn't send a deadline and it always has the default value.</summary>
```

Comment:
For ice1, I think it should be MaxValue, not default.

Reply:
Also, it should be sorted correctly - after Current.

Reply:
Fixed